coverage_badge/__main__.py

```diff
@@ -52,6 +64,8 @@ def parse_args(argv=None):
     parser = argparse.ArgumentParser(description=__doc__)
     parser.add_argument('-o', dest='filepath',
             help='Save the file to the specified path.')
+    parser.add_argument('-n', '--html', dest='html',                    # new
+            help='Get coverage from index.html output by nosetests.')   # new
```

Comment:
Sorry I haven't played with this much in a while. The project that was going to use this hasn't gotten enough attention lately and this is my first contribution to a project. Not off to a great start...  
No reason for binary mode to answer that question from way back, and I have no preferences on the -n vs -d options. Glad there's interest @pmourelle!